frontend/src/views/Quests.vue

```diff
@@ -2,10 +2,8 @@
   <div v-if="characters.length !== 0" class="d-flex flex-wrap quests-container gap-4">
     <div class="d-flex justify-content-between w-100 weekly-progress-container">
       <div class="d-flex flex-column justify-content-between gap-2">
-        <span class="quests-title">{{ $t('quests.quest') }}</span>
-        <b-button variant="primary" @click="showQuestsListModal = true">
-          {{ $t('quests.availableQuests') }}
-        </b-button>
+        <h5 class="quests-title">{{ $t('quests.quest') }}</h5>
+        <button-new class="custom-available-quest-button" :title="$t('quests.availableQuests')" @clickEvent="showQuestsListModal = true"></button-new>
```

Comment:
Can you rename this to like cb-button or something? Never use new in a name just delete the old one if it isn't being used